examples/prometheus/index.js

```diff
@@ -0,0 +1,25 @@
+const {Meter} = require("@opentelemetry/metrics");
+const {PrometheusExporter} = require("@opentelemetry/exporter-prometheus");
+
+const meter = new Meter();
+
+const exporter = new PrometheusExporter({
+  startServer: true,
+}, () => {
+  console.log("prometheus scrape endpoint: http://localhost:9464/metrics")
+})
+
+meter.addExporter(exporter);
+
+const monotonicCounter = meter.createCounter("monotonic_counter", {monotonic: true, description: "monotonic counter"});
+const nonmonotonicCounter = meter.createCounter("non_monotonic_counter", {monotonic: false, description: "non-monotonic counter"});
+const monotonicGauge = meter.createCounter("monotonic_gauge", {monotonic: true, description: "monotonic gauge"});
+const nonmonotonicGauge = meter.createCounter("non_monotonic_gauge", {monotonic: false, description: "non-monotonic gauge"});
+
+setInterval(() => {
+  monotonicCounter.getHandle(meter.labels({})).add(1);
```

Comment:
I think we should add some sample labels, something like `hostname` or `process.pid` or `method` or `code`.
Something like this:
```js
Const monotonicCounter = meter.createCounter("monotonic_counter", 
      {monotonic: true, 
       description: "Example of a monotonic counter", 
       labelKeys: ['code']
      });

// It is recommended to keep a reference to the Handle instead of always calling this 
// Method for every operation.
// const handle = monotonicCounter.getHandle(meter.labels({code: 200}));
// handle.add(1);
monotonicCounter.getHandle(meter.labels({code: 200})).add(1);
```

WDYT?